packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -1359,6 +1361,37 @@ export class AppStudioPluginImpl {
       appDefinition.outlineIcon = appManifest.icons.outline;
     }
 
+    appDefinition.configurableProperties = appManifest.configurableProperties;
```

Comment:
Does this affect project with old version manifest?

Reply:
it's optional, so older version without this field will just provide undefined.